HISTORY.rst

```diff
@@ -10,6 +10,7 @@ This project adheres to `Semantic Versioning <http://semver.org/>`__.
 Added
 ~~~~~
 - Values for Firefox Translations training repository scriptworker constants
+- Support for deferring upstream artifact selection until runtime
```

Comment:
I should have mentioned in the other PR, but this might warrant an update to the docs, at least cot_verify.rst, to mention globs.

Reply:
Good call; I will follow up with this.